convert.py

```diff
@@ -79,6 +80,17 @@ def get_static_data(modelSpec):
         return {}
 
 
+def labelize(propName):
+    """
+    If a propName doesn't match the 'simple-label' grammar, we return a quoted label
+    See: https://github.com/dhall-lang/dhall-lang/blob/1d2912067658fdbbc17696fc86f057d6f91712b9/standard/dhall.abnf#L125
+    """
+    if not re.match("^[a-zA-Z_][\w\-/]*$", propName):
```

Comment:
I think we need to use `[a-zA-Z0-9_]` instead of `\w`. The python documentation states  that `\w` matches any unicode string character but dhall does not allow them.

~~~python
import re
re.match("\w", "ö")
~~~

Reply:
Oh right, nice catch! Fixed in latest commit